src/PortAudio.jl

```diff
@@ -290,16 +303,16 @@ function PortAudioStream(fn::Function, args...; kwargs...)
 end
 
 function close(stream::PortAudioStream)
-    if stream.stream != C_NULL
-        Pa_StopStream(stream.stream)
-        Pa_CloseStream(stream.stream)
-        stream.stream = C_NULL
+    pointer = stream.pointer_ref[]
+    if stream.pointer_ref[] != C_NULL
+        Pa_StopStream(stream.pointer_ref[])
+        Pa_CloseStream(stream.pointer_ref[])
+        stream.pointer_ref[] = C_NULL
```

Comment:
It seems like we should just be able to set the pointer itself to 0? That would remove the need for a ref altogether. I followed the original code here.

Reply:
Oh, nevermind, I think I get why that wouldn't work. Pointers are immutable; only what they point to can change?